### ISSUE TITLE
Windows 10 Chrome post-download experiment.

### ISSUE THREAD

**Issue by amychurchwell** [mozilla/bedrock#8853]
## Description

![image](https://user-images.githubusercontent.com/30483988/80395084-96b9f600-8867-11ea-88a4-b819569f07cd.png)

Titled "/new chrome windows 10 post-download install instructions" in convert.

airtable card: https://airtable.com/tblO5zD4uNk7ISyG0/viwIUJsHKDrOFl90B/recUZVMgB8JzM3Zml

---

#### Test Plan

test Plan: https://docs.google.com/document/d/1O2e7_7aAOm982G291LJi8avydXNAkr_qjkyIeHhcB9U/edit# 

#### Audience
En-us, desktop, Chrome, Windows10 users (note: we are targeting /new users -- a percentage of which will go to /thanks)

**Comment by alexgibson** [comment]
The experiment code has the following custom JS:

```javascript
convert._$('LI.os_osx > A.download-link.button.button-green.mzp-c-button.mzp-t-product').replaceWith('<a class="download-link button button-green mzp-c-button mzp-t-product" href="/firefox/download/thanks/?v=b" data-direct-link="https://download.mozilla.org/?product=firefox-latest-ssl&amp;os=osx&amp;lang=en-US" data-link-type="download" data-display-name="macOS" data-download-version="osx" data-download-os="Desktop" data-download-location="primary cta">\n          <strong class="download-title">\n            \n              Download Now\n            \n          </strong>\n        </a>');
```

The `data-direct-link="https://download.mozilla.org/?product=firefox-latest-ssl&amp;os=osx&amp;lang=en-US"` part is pointing to the OSX version of the download.

I'd suggest either changing this to the Windows version (seeing as this experiment targets Windows only), or alternatively writing some JS to only append the parameter to the URL, as opposed to replace the button HTML.

**Comment by amychurchwell** [comment]
@alexgibson I updated to:
`
convert._$('LI.os_win > A.download-link.button.button-green.mzp-c-button.mzp-t-product').replaceWith('<a class="download-link button button-green mzp-c-button mzp-t-product" href="/firefox/download/thanks/?v=b" data-direct-link="https://download.mozilla.org/?product=firefox-stub&amp;os=win&amp;lang=en-US" data-link-type="download" data-display-name="Windows 32-bit" data-download-version="win" data-download-os="Desktop" data-download-location="primary cta">\n          <strong class="download-title">\n            \n              Download Now\n            \n          </strong>\n        </a>');
`

--> `data-direct-link="https://download.mozilla.org/?product=firefox-stub&amp;os=win&amp;lang=en-US"`

**Comment by amychurchwell** [comment]
note of relevance: hoping to launch this one on Thursday, when the brand nav experiment on /new concludes.

**Comment by alexgibson** [comment]
I re-enabled the test on staging and was able to enter the experiment using Chrome on Windows 10, and I saw the /thanks/ variation as expected. Things look good 👍 

I also tried the experiment on some other Chromium based browsers that masquerade as Chrome. 

- Brave blocks Convert as a tracker so we're OK there (as far as this experiment goes at least), but otherwise I believe those users would still see the Chrome /thanks/ instructions.
- Using Vivaldi, I did manage to enter the experiment and got the wrong experience:

![Screenshot (2)](https://user-images.githubusercontent.com/400117/80578600-9ef96900-8a00-11ea-9c02-1e957ffd8784.png)

I'm sure there are likely other Chromium browsers where this is an issue, but they are likely a small % of visits. There's a small chance it could potentially skew results still, but I'd say it's likely not a blocker. We'd need to tackle this properly should this experience ever get to production as a feature.
